diff -urN seahorse-plugins-2.24.0/m4/gecko.m4 seahorse-plugins-2.24.0.new/m4/gecko.m4
--- seahorse-plugins-2.24.0/m4/gecko.m4	2008-08-05 21:12:36.000000000 +0200
+++ seahorse-plugins-2.24.0.new/m4/gecko.m4	2008-10-08 19:54:37.000000000 +0200
@@ -100,7 +100,7 @@
 esac
 
 if test $gecko_cv_gecko = libxul-unstable; then
-_GECKO_INCLUDE_ROOT="`$PKG_CONFIG --variable=includedir ${gecko_cv_gecko}`/unstable"
+_GECKO_INCLUDE_ROOT="`$PKG_CONFIG --variable=includedir ${gecko_cv_gecko}`"
 _GECKO_HOME=""
 _GECKO_PREFIX="`$PKG_CONFIG --variable=prefix ${gecko_cv_gecko}`"
 else
diff -urN seahorse-plugins-2.24.0/plugins/epiphany/mozilla/Makefile.am seahorse-plugins-2.24.0.new/plugins/epiphany/mozilla/Makefile.am
--- seahorse-plugins-2.24.0/plugins/epiphany/mozilla/Makefile.am	2008-06-30 20:21:05.000000000 +0200
+++ seahorse-plugins-2.24.0.new/plugins/epiphany/mozilla/Makefile.am	2008-10-08 19:56:13.000000000 +0200
@@ -10,6 +10,7 @@
 	docshell	\
 	dom		\
 	gtkembedmoz	\
+	string		\
 	webbrwsr	\
 	widget		\
 	xpcom
